### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by RobLoach**

### CODE REVIEW

**Comment by RobLoach** [comment]
Strange that Travis isn't reporting.

**Comment by RobLoach** [comment]
> So the license was deleted?

Just didn't rebuild with PRojectz.

**Comment by Greduan** [comment]
I see. So if this is merged in then whoever packs it up next time has to build it with PRojectz?

**Comment by RobLoach** [comment]
> So if this is merged in then whoever packs it up next time has to build it with PRojectz?

I wouldn't consider it merge this merge ready yet. Stuck on a blocked label until projectz compile is run and a re-push. Thanks Greduan :+1:

File: package.json [hunk 1]
```diff
@@ -120,52 +121,52 @@
...
-    "coffee-script": "~1.8.0",
-    "chai": "~1.10.0",
-    "joe": "~1.5.0",
-    "joe-reporter-console": "~1.2.1",
-    "projectz": "~0.3.17"
+    "chai": "^1.10.0",
+    "coffee-script": "^1.8.0",
+    "joe": "^1.5.0",
+    "joe-reporter-console": "^1.2.1",
+    "projectz": "^0.3.17"
```
Comment (RobLoach) [inline package.json]: @balupton @Greduan Should we stick with using `~` for dependency declaration? I don't see a direct benefit out of using `^`. I just ran `npm update --save`.

Comment (Greduan) [inline package.json]: Myself I am not sure of the difference, usually I use whatever `--save` gives me...

Comment (balupton) [inline package.json]: We should keep `~`

**Comment by balupton** [comment]
Fixed in latest.

**Pull request closed without merging**